onnxscript/optimizer/_constant_folding.py

```diff
@@ -439,12 +439,16 @@ def split_to_sequence(node: ir.Node, op, state: OptimizerState) -> ReturnValue:
     else:
         return None
 
+    # If Split returns a single value, we need to wrap it into a list.
+    if isinstance(split_values, ir.Value):
+        split_values = [split_values]
```

Comment:
Looks like this line was not covered

Reply:
Will add test case later. It's triggered by a model in the benchmark. Will merge it to enable benchmarking. It's a weird example, though, and worth investigating more ... because a split of a tensor into one value is a no-op or identity. A candidate for optimization, either via rewrite or avoid generating it in the first-place.